app/controllers/ConfirmTransferController.scala

```diff
@@ -49,19 +50,33 @@ class ConfirmTransferController @Inject() (
       implicit request: Request[AnyContent]
   ): Future[Result] = {
     consignmentStatusService.getConsignmentStatuses(consignmentId, request.token.bearerAccessToken).flatMap { consignmentStatuses =>
+      val consignmentStatusValues: Map[Statuses.StatusType, Option[String]] =
+        consignmentStatusService.getStatusValues(consignmentStatuses, SeriesType, TransferAgreementType, UploadType, ClientChecksType)
       val exportTransferStatus = consignmentStatusService.getStatusValues(consignmentStatuses, ExportType).values.headOption.flatten
```

Comment:
Do you need to make the call to get the export transfer status separately? The `consignmentStatusService.getStatusValues` is called already. 

Could the export status value be included in this first call, and then retrieved from the resulting map? Would save a call to the DB.